web/satellite/src/plugins/analytics.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import VueSegmentAnalytics from 'vue-segment-analytics';
+
+import { isDoNotTrackEnabled } from '@/utils/doNotTrack';
+
+export const Analytics = {
+    install(Vue, options) {
+        const isDoNotTrack = isDoNotTrackEnabled();
+        const hasSegmentID = options.id && options.id.length > 0;
+
+        if (!hasSegmentID) {
+            options.id = 'fake id';
+        }
+
+        if ((isDoNotTrack || !hasSegmentID) && options.router !== undefined) {
+            delete options.router;
+        }
+
+        VueSegmentAnalytics.install(Vue, options);
+
+        // tslint:disable-next-line
+        if (isDoNotTrack || !hasSegmentID) {
+            Vue.$segment.methods.forEach(method => {
```

Comment:
I'm trying to understand why we remove the methods and we just not avoid to install the `SegmentAnalyitics` plugin under the above condition.